include/alpaka/mem/buf/omp5/Set.hpp

```diff
@@ -96,20 +97,23 @@ namespace alpaka
                 std::uint8_t const& byte,
                 TExtent const& /* extent */)
             {
-                using Idx = typename alpaka::trait::IdxType<TExtent>::type;
-                using Dim1D = DimInt<1u>;
-                using Vec1D = Vec<Dim1D, Idx>;
-
                 static_assert(Dim<TView>::value == 0u, "The view is required to have dimensionality 0!");
                 static_assert(Dim<TExtent>::value == 0u, "The extent is required to have dimensionality 0!");
 
-                return createTaskKernel<AccOmp5<Dim1D, Idx>>(
-                    WorkDivMembers<Dim1D, Idx>(Vec1D::zeros(), Vec1D::zeros(), Vec1D::zeros()),
-                    MemSetKernel(),
+                using ExtIdx = Idx<TExtent>;
+                using Dim0 = DimInt<0u>;
+                using Acc = AccOmp5<Dim0, ExtIdx>;
+                auto kernel = [] ALPAKA_FN_ACC(const Acc&, std::uint8_t b, std::uint8_t* mem)
+                {
+                    for(auto i = 0u; i < sizeof(Elem<TView>); i++)
```

Comment:
Please call the original kernel `MemSetKernel` because your for loop is not covering pitched memory.